recipes-core/images/adlink-image-validation.bb

```diff
@@ -0,0 +1,27 @@
+DESCRIPTION = "Adlink Image to validate Ampere machines,COM HPC - AVA/AP1 Platforms  \
+This image contains everything used to test COM HPC Platform , This creates a very large image, not \
+suitable for production."
+LICENSE = "MIT"
+
+inherit core-image
+
+WKS_FILE:comhpc = "adlink-efidisk.wks"
+
+## Select Image Features
+IMAGE_FEATURES += " \
+    debug-tweaks \
+    tools-profile \
+    nfs-server \
+    tools-debug \
+    ssh-server-dropbear \
+    tools-testapps \
```

Comment:
I'm just wondering what is the benefit of having tools-profile, tools-debug and tools-testapps in a validation image.

Reply:
This is part of standard Image requirement. we will remove after consulting concerned members